WordPress/Classes/Utility/PushNotificationsManager.swift

```diff
@@ -169,7 +169,8 @@ final public class PushNotificationsManager: NSObject {
         let handlers = [ handleSupportNotification,
```

Comment:
I know you didn't do it, but where did this bracket formatting come from 🤨

Reply:
As if you need to ask... (cough @jleandroperez)

Reply:
LOL doesn't it look neat!?!? Miss you all!!! <3

Reply:
Noooow... these days i'd write this same snippet as follows:

```
let handlers = [ 
    handleSupportNotification,
    handleAuthenticationNotification,
    handleInactiveNotification,
    handleBackgroundNotification,
    handleQuickStartLocalNotification
]
```

Progress perhaps?